lib/js/components/Banner/Banner.vue

```diff
@@ -0,0 +1,375 @@
+<template>
+	<div class="o-banner" :class="[colorClass, layoutClass]">
```

Comment:
`o-` prefix (meaning organism) is an artefact from atomic design which we no longer use. I'd rename it to `ds-banner`, per https://bethinkteam.slack.com/archives/CFBLF5RGA/p1636718839031100